### PR TITLE
WF-65091-Fix recurring formatting issues

### PR DESCRIPTION
**Pull request by sharonaumao**
<table style="width: 521.306px;">
<tbody>
<tr>
<td style="width: 156px;">Task Link</td>
<td style="width: 349.306px;">https://syncfusion.atlassian.net/browse/WF-65091</td>
</tr>
<tr>
<td style="width: 156px;">Description</td>
<td style="width: 349.306px;">Resolved formatting issues in Presentation UG</td>
</tr>
<tr>
<td style="width: 156px;">Solution</td>
<td style="width: 349.306px;">Resolve formatting and hyperlink issues in FAQ, Getting-Started, html, word-to-image, word-to-pdf</td>
</tr>
<tr>
<td style="width: 156px;">Content review confirmation</td>
<td style="width: 349.306px;">Yes</td>
</tr>
</tbody>
</table>

[WF-65091]: https://syncfusion.atlassian.net/browse/WF-65091?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Bookmarks.md [hunk 1]
```diff
@@ -9,7 +9,9 @@ documentation: UG
 
 A bookmark identifies a location or a selection of text within a document that you can name and identify for future reference.
 
-In Essential DocIO, bookmark is represented by Bookmark instance that is a pair of BookmarkStart and BookmarkEnd. BookmarkStart represents start point of a bookmark and BookmarkEnd represents end point of a bookmark. Every Word document contains a collection of bookmarks that are accessible through the Bookmarks property of WordDocument class.
+In Essential DocIO, bookmark is represented by [Bookmark](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.Bookmark.html) instance that is a pair of [BookmarkStart](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.BookmarkStart.html) and [BookmarkEnd](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.BookmarkEnd.html).
+
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply File-Formats/DocIO/Working-with-Bookmarks.md]: @Suriya-Balamurugan -The empty line is removed

File: File-Formats/DocIO/Working-with-Tables.md [hunk 2]
```diff
@@ -1101,7 +1101,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Apply formatting to Table, Row and Cell
 
-The following code example illustrates how to load an existing document and apply table formatting options such as Borders, LeftIndent, Paddings, IsAutoResize, etc.
+The following code example illustrates how to load an existing document and apply table formatting options such as [Borders](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.RowFormat.html#Syncfusion_DocIO_DLS_RowFormat_Borders), [IndentFromLeft](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.RowFormat.html#Syncfusion_DocIO_DLS_RowFormat_LeftIndent), [Paddings](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.RowFormat.html#Syncfusion_DocIO_DLS_RowFormat_Paddings), [IsAutoResized](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.RowFormat.html#Syncfusion_DocIO_DLS_RowFormat_IsAutoResized), etc.
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply File-Formats/DocIO/Working-with-Tables.md]: @Suriya-Balamurugan -Addressed

File: File-Formats/DocIO/mail-merge/mail-merge-events.md [hunk 3]
```diff
@@ -330,7 +330,7 @@ Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Template.docx", "appli
 
 {% endtabs %}
   
-The following code example shows how to bind the image from file system during Mail merge process by using MergeImageFieldEventHandler.
+The following code example shows how to bind the image from file system during Mail merge process by using [MergeImageFieldEventHandler](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.MergeFieldEventHandler.html).
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply File-Formats/DocIO/mail-merge/mail-merge-events.md]: @Suriya-Balamurugan -Feedback addressed

File: File-Formats/DocIO/mail-merge/mail-merge-for-group.md [hunk 4]
```diff
@@ -15,12 +15,13 @@ You can perform Mail merge and append multiple records from data source within a
 The region where the Mail merge operations are to be performed must be marked by two merge fields with the following names.
 
   * «TableStart:TableName» and «BeginGroup:GroupName» - For the entry point of the region.
-  
+
   * «TableEnd:TableName» and «EndGroup:GroupName» - For the end point of the region.
-  
-  1.*TableStart* and *TableEnd* region is preferred for performing Mail merge inside the table cell.  
+
+  1.*TableStart* and *TableEnd* region is preferred for performing Mail merge inside the table cell.
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply File-Formats/DocIO/mail-merge/mail-merge-for-group.md]: @Suriya-Balamurugan -Feedback addressed

File: File-Formats/DocIO/word-file-formats.md [hunk 5]
```diff
@@ -883,10 +883,10 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ### Open a Word (*.doc) document containing incremental save information
 
-Essential DocIO process the content that are preserved in the last complete save operation alone from a Word (.doc) document and it doesn't process the incremental save information. 
-Hence it throws "Complex format is not supported" exception when attempting to open a Word (.doc) document containing incremental save information.
+Essential DocIO process the content that are preserved in the last complete save operation alone from a Word (.doc) document and it doesn't process the incremental save information. Hence it throws "Complex format is not supported" exception when attempting to open a Word (.doc) document containing incremental save information.
+
+You can open the Word (*.doc) documents containing incremental save information without exception by setting [SkipIncrementalSaveValidation](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.Settings.html#Syncfusion_DocIO_DLS_Settings_SkipIncrementalSaveValidation) property of Settings class as true.
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply File-Formats/DocIO/word-file-formats.md]: @Suriya-Balamurugan -Feedback addressed

File: File-Formats/DocIO/word-to-pdf.md [hunk 6]
```diff
@@ -171,10 +171,16 @@ pdfDocument.Close();
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF).
 
 N> 1. Word to PDF conversion is not supported in Silverlight, Windows Phone, WinRT, and Universal applications.
-N> 2. For .NET Framework, creating an instance of the ChartToImageConverter class is mandatory to convert the charts present in the Word to PDF. Otherwise, the charts are not preserved in the converted PDF. Whereas this is not necessary for .NET Core, as ChartToImageConverter is initialized internally in Syncfusion.DocIORenderer.Portable assembly.
-N> 3. The ChartToImageConverter is supported from .NET Framework 4.0 onwards and .NET Core 2.0 onwards.
-N> 4. Total number of pages in the converted PDF may vary based on unsupported elements in the input Word document.
-N> 5. "DocIO supports Word to PDF conversion in UWP application using DocIORenderer." For further information, please refer [here](https://www.syncfusion.com/kb/10270/how-to-convert-word-document-to-pdf-in-uwp)
+
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply File-Formats/DocIO/word-to-pdf.md]: @Suriya-Balamurugan -Feedback addressed

File: File-Formats/Presentation/Loading-and-Saving-the-Presentation.md [hunk 7]
```diff
@@ -55,7 +55,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Opening an existing Presentation from stream
 
-You can open an existing PowerPoint Presentation from stream by using the overloads of Open method.
+You can open an existing PowerPoint Presentation from stream by using the overloads of [Open](https://help.syncfusion.com/cr/file-formats/Syncfusion.Presentation.Presentation.html#Syncfusion_Presentation_Presentation_Open_System_IO_Stream_System_String_) method.
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply File-Formats/Presentation/Loading-and-Saving-the-Presentation.md]: @Suriya-Balamurugan -Feedback addressed

File: File-Formats/Presentation/NuGet-Packages-Required.md [hunk 8]
```diff
@@ -91,6 +91,7 @@ Windows UI Library (WinUI) and .NET Multi-platform App UI (.NET MAUI)
 </table>
 
 N> 1.Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, add the "Syncfusion.Licensing" assembly reference and include a license key in your projects. Refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/license-key) to learn about registering Syncfusion license key in your applications to use the components.
+
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply File-Formats/Presentation/NuGet-Packages-Required.md]: @Suriya-Balamurugan -Feedback addressed

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**